examples/draft_transaction/draft_transaction.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"context"
+	"log"
+
+	wallet "github.com/bitcoin-sv/spv-wallet-go-client"
+	"github.com/bitcoin-sv/spv-wallet-go-client/commands"
+	"github.com/bitcoin-sv/spv-wallet-go-client/examples"
+	"github.com/bitcoin-sv/spv-wallet-go-client/examples/exampleutil"
+	"github.com/bitcoin-sv/spv-wallet/models/response"
+)
+
+func main() {
+	spv, err := wallet.NewWithXPriv(exampleutil.ExampleConfig, examples.XPriv)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	transaction, err := spv.DraftTransaction(context.Background(), &commands.DraftTransaction{
```

Comment:
IMHO instead of splitted examples for draft tx and record tx we should rather have (like in previous version) example for sending paymail transaction (it was in `create_transaction/`) and for creating op return tx (it was in `send_op_return/`)